{
  "Header": {
    "Original": "CVE-2025-38588 (In the Linux kernel, the following vulnerability has been resolved:  i ...)",
    "ID": "CVE-2025-38588",
    "Description": "(In the Linux kernel, the following vulnerability has been resolved:  i ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-4328-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-4328-1"
      ]
    },
    {
      "Original": "- linux 6.16.3-1",
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.16.3-1"
    },
    {
      "Original": "[trixie] - linux 6.12.43-1",
      "Type": "package",
      "Release": "trixie",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.12.43-1"
    },
    {
      "Original": "[bookworm] - linux 6.1.148-1",
      "Type": "package",
      "Release": "bookworm",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "6.1.148-1"
    },
    {
      "Original": "[bullseye] - linux \u003cnot-affected\u003e (Vulnerable code not present)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "linux",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/54e6fe9dd3b0e7c481c2228782c9494d653546da (6.17-rc1)",
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/54e6fe9dd3b0e7c481c2228782c9494d653546da (6.17-rc1)"
    }
  ]
}